docs/kafka/topic-configuration-kafka/README.adoc

```diff
@@ -783,55 +781,54 @@ h|Kafka property name
 
-For a given Kafka topic, you can use the console to check {registry} instances for value or key schemas registered to those instances that match the name of the topic. If you don't have access to any {registry} instances, or don't have value or key schemas registered to your instances that match your topic, the console provides links to {product-registry}, where you can create instances and schemas. The console also shows you the naming format you need to use when creating a new value or key schema, so that it matches the topic.
+For a given Kafka topic, you can use the console to check {registry} instances for value or key schemas registered to those instances that match the name of the topic. If you do not have access to any {registry} instances, or you do not have value or key schemas registered to your instances that match your topic, the console provides links to {product-registry}, where you can create instances and schemas. The console also shows you the naming format you need to use when creating a new value or key schema, so that it matches the topic.
 
 [id="proc-checking-topic-for-existing-schema-matches_{context}"]
 === Checking a topic for existing schema matches
 
 [role="_abstract"]
-The following procedure shows how to use the web console to select a Kafka topic and then check an existing {registry} instance for value or key schemas that have IDs that match the name of the topic.
+The following procedure shows how to use the {product-long-kafka} web console to select a Kafka topic and then check an existing {product-long-registry} instance for value or key schemas that have IDs that match the name of the topic.
 
 Alternatively, to learn how to create a _new_ {registry} instance with a value or key schema that matches a topic, see {base-url}{topic-config-url-kafka}#proc-creating-registry-instance-and-matching-schema-for-topic_{context}[Creating a new registry instance and matching schema for a topic].
 
 .Prerequisites
-* You're logged in to the {product-kafka} web console.
-* You've created a Kafka instance with at least one topic in {product-kafka}. To learn how to do this, see {base-url}{getting-started-url-kafka}[Getting started with {product-long-kafka}^].
+* You have created a Kafka instance with at least one topic in {product-kafka}. To learn how to do this, see {base-url}{getting-started-url-kafka}[Getting started with {product-long-kafka}^].
 * You understand how to create a {registry} instance and upload a schema to be used by client applications. To learn how to do this, see {base-url}{getting-started-url-registry}[Getting started with {product-long-registry}^].
 * You have access to at least one instance in {registry} that you can check for schemas that match your topic.
 
 .Procedure
-. In the {product-kafka} web console, go to *Streams for Apache Kafka* > *Kafka Instances*. Click the name of the Kafka instance that contains the topic that you want to check for matching schemas in {product-registry}.
```

Comment:
Surely I didn't write "go to"!? :roll_eyes: Thanks for fixing :relaxed:

Reply:
Oh, that's not you, it's actually more me, and it was intentional. I bet you copied the standard we/I established early on. We say "go to" everywhere in docs today where we have a path for users to follow, including in my docs. As opposed to "navigate to" of just "select X > Y > Z". IBM says to just say "select", "click", "expand", whatever, per [Menus and navigation](https://www.ibm.com/docs/en/ibm-style?topic=elements-menus-navigation) section. 

However, the deviation to go with "go to" was intentional, because we found on my previous team (BA) that IBM's ruling of "select", "click", "expand" doesn't always work with navigations because the first thing in the series could be something you click, the second something you select, and the third something you expand. So picking one verb for the navigation series just wasn't scalable or sensical many times. Whereas "navigate to" or "go to" works every time. So we adopted that going forward years ago, and I guess I brought that with me :)

That said, this standard was admittedly brought into play mostly by me, and I think Ben followed, then everyone else likewise followed, etc. So it sounds like we should revisit it and my logic above for it as a team and decide whether to keep doing it or ditch it and add it to that style table checklist of things to rectify.

Reply:
Ah, I'm sorry for picking on this phrase specifically - I was sure I was the one to have written it. I actually forgot how much it crops up elsewhere. 

When I read it yesterday, it suddenly jumped out to me as very vague in the action it's communicating to users. But the logic you described in your comment makes total sense. I'd even say that it suits the informal voice of our quick starts, where "click", "select", etc. get quite stuffy.

So we'll maybe take a look at this again if we have more style discussions, but I definitely don't see it as a burning issue. Thanks for taking the time to explain the origin of this.

Reply:
Ack, thanks.